### PR TITLE
Add brand and change alignment

### PR DESCRIPTION
**Pull request by ironyee**

### CODE REVIEW

**Comment by otabek024** [comment]
here we will discuss?

File: index.html [hunk 1]
```diff
@@ -30,7 +30,11 @@ <h5 class="text-white h4">Contact</h5>
     </div>
   </div>
   <nav class="navbar navbar-dark bg-dark">
-    <div class="container-fluid d-flex justify-content-end">
```
Comment (ironyee) [inline index.html:33]: Do you know what justify-content property is?
Why did you use justify-content-end class?

Comment (otabek024) [reply index.html:33]: yes bro i know first i tried justify-content space between but did not work

Comment (otabek024) [reply index.html:33]: then I wrote justify cotent end in order to move navbar toggler to right side

Comment (otabek024) [reply index.html:33]: it is working now after you have changed it

Comment (ironyee) [reply index.html:33]: To put the brand on the left side and the menu button on the right side, you need to use justify-content-space.
https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content

If it didn't work, you must have made another mistake.

Comment (otabek024) [reply index.html:33]: now we need to change 9 mins to right side?

Comment (ironyee) [reply index.html:33]: Okay. Merge this pull request first.

Comment (otabek024) [reply index.html:33]: for me it shows same or did you fix it?

**Pull request merged**